docs/advisories/README.md

```diff
@@ -1,7 +1,8 @@
 ## kops Advisories
 
-- [Meltdown and Spectre](meltdown-spectre.md)
+- [Meltdown Kernel Update Required](meltdown-kernel-update.md)
```

Comment:
I am confused, as I am reading that one spectre cve and one meltdown cve was fixed.

From https://googleprojectzero.blogspot.co.uk/2018/01/reading-privileged-memory-with-side.html

So far, there are three known variants of the issue:

Variant 1: bounds check bypass (CVE-2017-5753)
Variant 2: branch target injection (CVE-2017-5715)
Variant 3: rogue data cache load (CVE-2017-5754)

Before the issues described here were publicly disclosed, Daniel Gruss, Moritz Lipp, Yuval Yarom, Paul Kocher, Daniel Genkin, Michael Schwarz, Mike Hamburg, Stefan Mangard, Thomas Prescher and Werner Haas also reported them; their [writeups/blogposts/paper drafts] are at:

Spectre (variants 1 and 2)
Meltdown (variant 3)

Reply:
Grrr... I think you're right.  I'll tweak - I'm just calling to call them spectre/meltdown.  The separate CVEs are just muddying the waters IMO